README.md

```diff
@@ -28,7 +28,23 @@ ecs_clusters:                                     # the list of ECS cluster conf
       - us-east-1a
     subnets:                                      # List of subnets
       - subnet-abcd1234
+    cluster_type: ec2                             # Cluster type: ec2 or fargate
+
```

Comment:
It could be clear if some comments here, like: "To create an fargate cluster you just run...."

Reply:
done!